### DISCUSSION TITLE
Raycast3D - Collide with areas default ON

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9991] [Editor]
Raycast3d node, collide_with_area is defaulted to OFF.

This means everytime you tell someone to make a raycast3D node, you have to remind them to tick this box then again when they tell you it doesn't work.

Every. Time.

Even the docs need to remind users!: "Note that to enable collisions with Area3D, the boolean parameter collide_with_areas must be set to true."

The default need to change. It's boolean, so default it to ON. 

Thank you.

**Comment by AThousandShips** [comment]
This breaks compatibility so can't be done without breaking a lot of existing projects

**Reply by shak2** [reply]
4.4 or 5.0.

Book it in. Let's go!

**Reply by AThousandShips** [reply]
5.0 if at all, not 4.4, and 5.0 is years away

**Reply by AThousandShips** [reply]
I'd suggest opening a proposal to track this and more details and see about what support it gets

**Comment by jmarceno** [comment]
I have a pretty sizable 3d project and I don't think I have a single raycast that is setup to collide with an area.

They all collide with colliders.
I only use areas to receive signals for other bodies and areas 

But maybe I'm the odd one here.
